101-1vm-2nics-2subnets-1vnet/azuredeploy.json

```diff
@@ -124,12 +124,25 @@
       "kind": "Storage",
       "properties": {}
     },
+    {
+      "comments": "Simple Network Security Group for subnet [variables('subnet2Name')]",
+      "type": "Microsoft.Network/networkSecurityGroups",
+      "apiVersion": "2019-08-01",
+      "name": "[concat(variables('subnet2Name'), '-nsg')]",
```

Comment:
create a variable for the nsg name since we use it in many places... can be something like

"publicNSG": "[concat(var(subnetname), -nsg)]"

Or any var name that describes what we're providing.

Reply:
Added variable.